### PR TITLE
Cloudposse master

### PR DESCRIPTION
**Pull request by rverma-jm**
## what
* Describe high-level what changed as a result of these commits (i.e. in plain-english, what do these changes mean?)
* Use bullet points to be concise and to the point.

## why
* Provide the justifications for the changes (e.g. business case). 
* Describe why these changes were made (e.g. why do these commits fix the problem?)
* Use bullet points to be concise and to the point.

## references
* Link to any supporting github issues or helpful documentation to add some context (e.g. stackoverflow). 
* Use `closes #123`, if this PR closes a GitHub issue `#123`

### CODE REVIEW

**Comment by joe-niland** [comment]
@rverma-jm thanks for the PR. Please update PR title and description so it's clear what it's for.

**Comment by aknysh** [comment]
@rverma-jm  thanks for the PR, please update the `what` and `why` so we understand what is being changed and why

File: main.tf [hunk 1]
```diff
@@ -44,7 +44,14 @@ resource "aws_security_group_rule" "egress" {
   security_group_id = join("", aws_security_group.default.*.id)
 }
 
-resource "aws_rds_cluster" "primary" {
```
Comment (jamengual) [inline main.tf:47]: Changing the resource name will destroy every single cluster already created using this module, this is a not needed breaking change.

File: main.tf [hunk 2]
```diff
@@ -44,7 +44,14 @@ resource "aws_security_group_rule" "egress" {
...
+resource "aws_cloudwatch_log_group" "default" {
+  count             = var.enabled && length(var.enabled_cloudwatch_logs_exports)>0 ? 1 : 0
+  name              = "/aws/rds/cluster/${coalesce(join("", aws_rds_cluster.default.*.id), join("", aws_rds_cluster.secondary.*.id))}/postgresql"
```
Comment (Nuru) [inline main.tf:49]: I am not completely sure how log groups work with primary and secondary clusters. According to [this documentation](https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/AuroraPostgreSQL.CloudWatch.html#AuroraPostgreSQL.CloudWatch.Monitor) you need a log group per log type anyway.

```suggestion
  for_each          = module.this.enabled ? toset(var.enabled_cloudwatch_logs_exports) : []
  name              = "/aws/rds/cluster/${coalesce(join("", aws_rds_cluster.primary.*.id), join("", aws_rds_cluster.secondary.*.id))}/${each.value}"
```

Also, update description of `var.enabled_cloudwatch_logs_exports` to include `postgresql` in the list of valid log types.

**Review by Nuru — CHANGES_REQUESTED** [review]
- Fill in the PR description
- Restore the "primary" name. 
- Validate that the log groups work as expected with all database engine types
- Make other changes as noted in comments

**Comment by Nuru** [comment]
Closing this as unclear on the purpose and apparently abandoned by the OP. Can be reopened if `what` and `why` are filled in and other comments addressed.

**Pull request closed without merging**